Automattic/wp-calypso
# Performance Profiler: update history chart styling

## Proposed Changes

- center history chart info box
- remove extra left padding
- add corner radius to red squares
- format dates on the X axis
- show fewer datapoints on mobile

## Why are these changes being made?


pdt2If-1Jh-p2

## Testing Instructions



* Go to `/speed-test` and start a new test, or go to a results page eg. `/speed-test-tool?url=https%3A%2F%2Fwordpress.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzA1N30.7KXaBL1qcm0y0GqvM8ymJ0pbST9ls6P09_hri8EaKSI`
* Check the styling of the history chart

| Before | After |
|--------|--------|
| ![CleanShot 2024-09-12 at 16 19 46@2x](https://github.com/user-attachments/assets/1ddf0ce8-573f-42de-8d84-c4847f8f1c6d) | ![CleanShot 2024-09-12 at 16 19 52@2x](https://github.com/user-attachments/assets/094e5d51-4ade-4ff9-b606-718a81bdc19d) |
| ![CleanShot 2024-09-12 at 16 21 18@2x](https://github.com/user-attachments/assets/1c704f8d-463c-4144-bc82-1e06c48cdcda) | ![CleanShot 2024-09-12 at 16 21 24@2x](https://github.com/user-attachments/assets/c43f8c4d-5c8f-4067-8fcd-6155c25e21e4) | 
| ![CleanShot 2024-09-12 at 16 28 31@2x](https://github.com/user-attachments/assets/29c6dbd4-1ea8-442b-be43-056903e359a1) | ![CleanShot 2024-09-12 at 16 28 48@2x](https://github.com/user-attachments/assets/736a14ce-d43f-47da-adbf-991f543a7d4e) |
